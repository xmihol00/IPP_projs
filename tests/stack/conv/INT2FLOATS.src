.IPPcode21

DEFVAR GF@num1
DEFVAR GF@num2

PUSHS int@26
INT2FLOATS
POPS GF@num1
WRITE GF@num1
WRITE string@\010

MOVE GF@num2 float@0x1.4dp+6
ADD GF@num2 GF@num1 GF@num2
WRITE GF@num2
WRITE string@\010

MOVE GF@num1 int@11
PUSHS GF@num1
INT2FLOATS
POPS GF@num1
WRITE GF@num1
WRITE string@\010
