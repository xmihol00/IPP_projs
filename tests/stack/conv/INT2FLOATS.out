0x1.a000000000000p+4
0x1.b500000000000p+6
0x1.6000000000000p+3
